## Adam Taranto

I'm a research software engineer, genome biologist, and educator who helps researchers of all levels
improve their computer science and data science skills.

#### üë©‚Äçüíª What I'm currently working on

- [Adamtaranto/mimeo](https://github.com/Adamtaranto/mimeo) - Scan genomes for internally repeated sequences, elements which are repetitive in another species, or high-identity HGT candidate regions between species. (today)
- [Adamtaranto/deRIP2](https://github.com/Adamtaranto/deRIP2) - Reconstruct ancestral state sequences of fungal repeat families by correcting for RIP-like mutations. Mask RIP or deamination events from alignments. (1 day ago)
- [Adamtaranto/tSplit](https://github.com/Adamtaranto/tSplit) - Extract terminal repeats from retrotransposons (LTRs) or DNA transposons (TIRs). Compose synthetic MITES from complete DNA transposons. (1 day ago)
- [WEHI-Education/Workshop-intermediate-slurm](https://github.com/WEHI-Education/Workshop-intermediate-slurm) - Intermedate Slurm Workshop source code using the Software Carpentry Workbench style. (5 days ago)
- [WEHI-Education/CarpentriesSlides](https://github.com/WEHI-Education/CarpentriesSlides) - Converts Carpentries workbench workshops to slides (5 days ago)

#### üî≠ Latest releases I've contributed to

- [Adamtaranto/mimeo](https://github.com/Adamtaranto/mimeo) ([v1.2.0](https://github.com/Adamtaranto/mimeo/releases/tag/v1.2.0), today) - Scan genomes for internally repeated sequences, elements which are repetitive in another species, or high-identity HGT candidate regions between species.
- [Adamtaranto/tSplit](https://github.com/Adamtaranto/tSplit) ([v0.1.4](https://github.com/Adamtaranto/tSplit/releases/tag/v0.1.4), 1 day ago) - Extract terminal repeats from retrotransposons (LTRs) or DNA transposons (TIRs). Compose synthetic MITES from complete DNA transposons.
- [Adamtaranto/deRIP2](https://github.com/Adamtaranto/deRIP2) ([v0.4.0](https://github.com/Adamtaranto/deRIP2/releases/tag/v0.4.0), 2 days ago) - Reconstruct ancestral state sequences of fungal repeat families by correcting for RIP-like mutations. Mask RIP or deamination events from alignments.
- [sanger-pathogens/Fastaq](https://github.com/sanger-pathogens/Fastaq) ([3.18.0](https://github.com/sanger-pathogens/Fastaq/releases/tag/3.18.0), 1 week ago) - Python3 scripts to manipulate FASTA and FASTQ files
- [sanger-pathogens/pymummer](https://github.com/sanger-pathogens/pymummer) ([0.12.0](https://github.com/sanger-pathogens/pymummer/releases/tag/0.12.0), 1 week ago) - Python3 module for running MUMmer and reading the output

<table>
  <tr style="border: none">
    <td valign="top" style="border: none">
      <img src="https://github-readme-stats.vercel.app/api?username=adamtaranto&rank_icon=percentile&show_icons=true&theme=transparent" alt="GitHub Stats" />
    </td>
    <td valign="top" style="border: none">
      <img src="https://github-readme-streak-stats.herokuapp.com?user=adamtaranto&mode=weekly&theme=transparent" alt="GitHub Streak" />
    </td>
  </tr>
</table>