## Adam Taranto

I'm a research software engineer, genome biologist, and educator who helps researchers of all levels
improve their computer science and data science skills.

#### üë©‚Äçüíª What I'm currently working on

- [Adamtaranto/teloclip](https://github.com/Adamtaranto/teloclip) -  A tool for the recovery of unassembled telomeres from soft-clipped read alignments. (3 days ago)
- [Adamtaranto/deRIP2](https://github.com/Adamtaranto/deRIP2) - Predict progenitor sequence of fungal repeat families by correcting for RIP-like mutations and cytosine deamination events. Mask RIP or deamination events from alignments. (3 days ago)
- [Adamtaranto/TIRmite](https://github.com/Adamtaranto/TIRmite) - Map TIR-pHMM models to genomic sequences for annotation of MITES and complete DNA-Transposons. (3 days ago)
- [oxli-bio/oxli](https://github.com/oxli-bio/oxli) - k-mers and the like (2 weeks ago)
- [WEHI-Education/Workshop-intermediate-slurm](https://github.com/WEHI-Education/Workshop-intermediate-slurm) - Intermedate Slurm Workshop source code using the Software Carpentry Workbench style. (3 weeks ago)

#### üî≠ Latest releases I've contributed to

- [sanger-pathogens/Fastaq](https://github.com/sanger-pathogens/Fastaq) ([3.18.0](https://github.com/sanger-pathogens/Fastaq/releases/tag/3.18.0), 1 day ago) - Python3 scripts to manipulate FASTA and FASTQ files
- [sanger-pathogens/pymummer](https://github.com/sanger-pathogens/pymummer) ([0.12.0](https://github.com/sanger-pathogens/pymummer/releases/tag/0.12.0), 1 day ago) - Python3 module for running MUMmer and reading the output
- [Adamtaranto/teloclip](https://github.com/Adamtaranto/teloclip) ([v0.1.1](https://github.com/Adamtaranto/teloclip/releases/tag/v0.1.1), 3 days ago) -  A tool for the recovery of unassembled telomeres from soft-clipped read alignments.
- [Adamtaranto/deRIP2](https://github.com/Adamtaranto/deRIP2) ([v0.3.0](https://github.com/Adamtaranto/deRIP2/releases/tag/v0.3.0), 4 days ago) - Predict progenitor sequence of fungal repeat families by correcting for RIP-like mutations and cytosine deamination events. Mask RIP or deamination events from alignments.
- [Adamtaranto/learn-hatch](https://github.com/Adamtaranto/learn-hatch) ([0.1.0](https://github.com/Adamtaranto/learn-hatch/releases/tag/0.1.0), 4 months ago) - Learning how to do python packaging with Hatch.

<table>
  <tr style="border: none">
    <td valign="top" style="border: none">
      <img src="https://github-readme-stats.vercel.app/api?username=adamtaranto&rank_icon=percentile&show_icons=true&theme=transparent" alt="GitHub Stats" />
    </td>
    <td valign="top" style="border: none">
      <img src="https://github-readme-streak-stats.herokuapp.com?user=adamtaranto&mode=weekly&theme=transparent" alt="GitHub Streak" />
    </td>
  </tr>
</table>